code/game/machinery/Sleeper.dm

```diff
@@ -412,6 +463,7 @@
 		go_out()
 	..(severity)
 
+// ???

```

Comment:
I'm still unsure what the section below this is supposed to do or mean; I can only assume it was something that ran periodically on old sleepers to keep patients up to health. Though, some of the conditionals there won't ever come true.